shell/pom.xml

```diff
@@ -69,5 +69,15 @@
       <artifactId>alluxio-core-common</artifactId>
       <version>${project.version}</version>
     </dependency>
+      <dependency>
```

Comment:
two comments:

1) you should be using `${project.version}` instead of `1.6.0-SNAPSHOT`

2) it seems that these dependencies are only needed to get that Alluxio Master / Worker class name ... if that's the case, I would prefer not to create this dependency and use a string constant to represent the class name